### ISSUE TITLE
Default all searches to the TPDB movies

### ISSUE THREAD

**Issue by thedesi1** [ThePornDatabase/namer#161]
Enhancement to namer so that a flag is set in config file and all the queries go to TPDB movies vs trying to also search Scenes

**Comment by thedesi1** [comment]
This is already implemented in the JF TPDB plugin

**Comment by 4c0d3r** [comment]
This will have to work differently.   

Namer needs another regex parser for movie formats....  parse as scene first, if that fails parse as movie (name - year.ext) -- ignore matching on site.

Namer likely should decide which data source to query first based which parser succeeds.     The jellyfin plugin likely should be left to match scenes first, and then movies if you are using namer's default name output pattern of site - data - name.   Which I don't think you are.

But keep in mind it's not about just your use case, but other's as well -- and it's up in the air if we should force movies and scenes to be in different libraries in jelly/plex or in one.   I happen to keep them all in one folder.

**Comment by thedesi1** [comment]
I agree, from my experience in Stash world, most folks are keeping them separate and I am asking to use default which is the current scenario or option to separate it like JF, which works perfectly fine. Just for the reference, 98% of the movies that are collected from scene dont match today and whatever match is incorrect, I have tested it with 1000+ titles.

**Comment by thedesi1** [comment]
also the manual mode that went with #170 will work for my use case so not very important to implement this.

**Comment by 4c0d3r** [comment]
The matching may work if  "name - year.ext" searching were a thing.    Glad 170 eases your current needs, but if namer ever supports movies fully I'd want to do it right.   Not sure what that means yet.   Is "name - year.ext" really that common?   Or is site-date-name.ext more common?  Should all the plugins be updated to support "name-year.ext", or should namer always output sites-date-name.ext instead (so the plugins just work)?   I don't know if we know the right answers yet.

**Comment by geek-baba** [comment]
You are right, I have been trying to figure out myself, each movie has multiple years and multiple studios, exactly same movie, tpdb needs to dedupe and do something like what I proposed in discord:
1.  One Movie - Multiple Studios
2. One Movie - Multiple release dates by Studios

**Comment by thedesi1** [comment]
You are right, I have been trying to figure out myself, each movie has multiple years and multiple studios, exactly same movie, tpdb needs to dedupe and do something like what I proposed in discord:

One Movie - Multiple Studios
One Movie - Multiple release dates by Studios

**Comment by thedesi1** [comment]
What's been working 99% time is this for me:
`name} ({year})/{name} ({year}) - {site} - {date} - {resolution}.{ext}`
